boards/aerotenna/ocpoc/ubuntu.cmake

```diff
@@ -4,7 +4,7 @@ px4_add_board(
 	MODEL ocpoc
 	LABEL ubuntu
 	PLATFORM posix
-	ARCH cortex-a9
+	PROCESSOR cortex-a9
```

Comment:
@dagar 

Is this the correct name? Is this going to be -mcpu or -march?

Reply:
It's used by the Toolchain to set the cpu flags, which usually sets both -mcpu and -march. We can expand it later if we need more granularity, but so far this covers everything we use without getting into gcc flag minutia.

https://github.com/PX4/Firmware/tree/pr-boards/platforms/nuttx/cmake/Platform

That reminds me that we shouldn't be setting both -mcpu and -march for arm.

Reply:
Changed to ARCHITECTURE